src/components/views/messages/TextualBody.tsx

```diff
@@ -509,17 +510,44 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
         }
         const mxEvent = this.props.mxEvent;
         const content = mxEvent.getContent();
+        let isNotice = false;
```

Comment:
~I think these can be limited to their separate scopes so line 523 would be `let isNotice =...` and so would line 541.  This would make it slightly easier for me to read.~
Nope, just found where you use them :-)

Reply:
~(same for isEmote, obviously)~